containerd/metadata.csv

```diff
@@ -22,6 +22,9 @@ containerd.mem.cache,gauge,,byte,,The cache amount used,0,containerd, cache_amou
 containerd.mem.rss,gauge,,byte,,The rss amount used,0,containerd, mem_rss
 containerd.mem.rsshuge,gauge,,byte,,The rss_huge amount used,0,containerd, mem_rss_huge
 containerd.mem.dirty,gauge,,byte,,The dirty amount,0,containerd, mem_dirty
+containerd.mem.commit,gauge,,byte,,The committed memory,0,containerd,mem_commit
+containerd.mem.commit_peak,gauge,,byte,,The peak committed memory,0,containerd,mem_commit_peak
+containerd.mem.private_working_set,gauge,,byte,,The private working set usage,0,containerd,mem_private_working_set
```

Comment:
could you add in the description of the different metrics `(windows only)`
like this it will be present in the application, like in the `metric summary` page

Reply:
if you can add it to, the linux only metrics 🙇

Reply:
Added :+1: